scalafix-tests/src/it/scala/scalafix/tests/URLConfiguration.scala

```diff
@@ -20,4 +20,37 @@ class URLConfiguration
        |
     """.stripMargin
   DiffTest("url", code, "UrlRewrite.stat").foreach(runDiffTest)
+
+  val githubURL =
+    "github:scalacenter/scalafix/1.0"
+  val githubCode =
+    s"""
+       |rewrites = [
+       |  "$githubURL"
+       |]
+       |<<< NOWRAP GitHub shorthand
```

Comment:
Since we only need to test the String => URL expansion, there's no need to test this.